risc0/zkvm/sdk/cpp/host/c_api.h

```diff
@@ -60,7 +60,8 @@ void risc0_init();
 // MethodId
 //
 
-risc0_method_id* risc0_method_id_new(risc0_error* err, const char* elf_path, uint32_t limit);
+risc0_method_id*
+risc0_method_id_new(risc0_error* err, const uint8_t* bytes, size_t len, uint32_t limit);
```

Comment:
There seems to be a mismatch, I see `risc0_method_id_compute` in the .cpp

Reply:
Fixed that one and a couple others, added build flag to enforce matching prototypes.